grafana/risingwave-dev-dashboard.dashboard.py

```diff
@@ -1591,6 +1591,38 @@ def section_batch(outer_panels):
                         ),
                     ],
                 ),
+
+                panels.timeseries_bytes(
+            "Mem Table Size",
```

Comment:
Can you please help to revert back the metric `Mem Table Size (Max)` in L2215:

```
-                    "Mem Table Size (Max)",
-                    "Write Btach Size",
```

And add a `avg` for it as well. Thanks a lot!